plugins/source/gcp/client/services.go

```diff
@@ -0,0 +1,63 @@
+package client
+
+var GcpServices = map[string]bool{
+	"apikeys.googleapis.com":               true,
```

Comment:
Can we make those consts and then add this to the array. Similar how this is done https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/client/namespaces.go

Reply:
I don't think that `const` would make any sort of improvement because we are still just dealing with strings in the codegen recipes...

In a future PR I would like to autogenerate this whole map like we do with the AWS data json file